CONTRIBUTING.md

```diff
@@ -10,7 +10,6 @@ Any new implementation is expected to meet the following standards before it can
 -   The component has a [**displayName**](#display-names) defined.
 -   The implementation complies with [**accessibility**](#accessibility) standards.
 -   The component is **properly exported** in `src/index.js` (maintain alphabetical order).
--   The implementation is thoroughly [**covered by tests**](#testing) (Jest / Enzyme, preferably 100%), not only with snapshots.
```

Comment:
This line still makes sense, just replace `Jest / Enzyme` with `Jest`

Reply:
:+1: